iteration/iteration_test.go

```diff
@@ -432,3 +455,88 @@ func (s *TestIterationRepository) TestExistsIteration() {
 	})
 
 }
+
+func (s *TestIterationRepository) TestIsActive() {
+	t := s.T()
+	t.Run("user active is true", func(t *testing.T) {
+		// given
+		newSpace := space.Space{
+			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
+			OwnerId: s.testIdentity.ID,
+		}
+		repoSpace := space.NewRepository(s.DB)
+		space, err := repoSpace.Create(context.Background(), &newSpace)
+		assert.Nil(t, err)
```

Comment:
You can move this setup code to create a space above all sub tests and then the subtests can create iterations inside of that space, no? So move these lines above all `t.Run(...)` calls:

```go
		newSpace := space.Space{
 			Name:    testsupport.CreateRandomValidTestName("Space Exists"),
 			OwnerId: s.testIdentity.ID,
 		}
 		repoSpace := space.NewRepository(s.DB)
 		space, err := repoSpace.Create(context.Background(), &newSpace)
 		assert.Nil(t, err)
```

Reply:
@kwk i will remove the redundant code and put it above t.Run(..)

Reply:
handled it here -> 6196068e5054df9bb9044b173546621bf9f3ce0c